Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/DriverInformation.hs

```diff
@@ -61,6 +65,16 @@ data DriverAutoPayStatus
   deriving (Show, Eq, Ord, Read, Generic, ToJSON, FromJSON, ToSchema, ToParamSchema)
   deriving (PrettyShow) via Showable DriverAutoPayStatus
 
+instance HasSqlValueSyntax be String => HasSqlValueSyntax be DriverAutoPayStatus where
```

Comment:
Why are we having this change in every file? It seems very redundant.

Reply:
No @hkmangla if you see in each file we are defining this instance for different type here for say DriverAutoPayStatus

Reply:
But this will become very redundant. Why it was not required before?

Reply:
@hkmangla we can try to reduce this boilerplate, like create some custom class `BeamRow` with default instance instead of these four `HasSqlValueSyntax`, `B.HasSqlEqualityCheck,` `FromBackendRow`, `FromField`
I can think about it in separate PR, if it make sense

Reply:
Sure @roman-bodavskiy . Lets not merge this one and do it properly.

Reply:
Yeah @hemantmangla  the plan was to move to TH and write instances. But we wanted to make sure things are working fine first